shell.nix

```diff
@@ -0,0 +1,41 @@
+{ pkgs ? import <nixpkgs> {} }:
+
+with builtins;
+let
+  inherit (pkgs) stdenv;
+  pythonPackages = stdenv.lib.fix' (self: with self; pkgs.python3Packages //
+  {
+
+    tockloader = buildPythonPackage rec {
+      pname = "tockloader";
+      version = "1.1.0";
+      name = "${pname}-${version}";
+
+      propagatedBuildInputs = [ argcomplete colorama crcmod pyserial pytoml ];
```

Comment:
Is this really just a copy of https://github.com/tock/tockloader/blob/master/setup.py#L31 ?

There's no way to pull that automatically from the package itself? :/

Reply:
Pretty much. I don't know nix shell well enough yet to know if there is a way to avoid doing that, but I needed it to build.